vcpkg-init/mint-standalone-bundle.ps1

```diff
@@ -72,16 +81,23 @@ try {
     }
 
     $bundleConfig = @{
-        'readonly' = $False;
+        'readonly' = $readonly.isPresent;
         'usegitregistry' = $True;
         'embeddedsha' = $sha
     }
+    New-Item -Path "out/.vcpkg-root" -ItemType "File"
 
     Set-Content -Path "out/vcpkg-bundle.json" `
         -Value (ConvertTo-Json -InputObject $bundleConfig) `
         -Encoding Ascii
-
-    tar czf $DestinationTarballName -C out *
+    if ($DestinationDir) {
+        if (-not (Test-Path $DestinationDir)) {
+            mkdir $DestinationDir
+        }
+        Copy-Item -Recurse out/* $DestinationDir

```

Comment:
Is this different than `Copy-Item -Recurse out $DestinationDir` ?

Instead of making tempdir and destinationdir different, can you merge the parameters and just have an option to skip the tar step?